MovieApp/Screens/Home/HomeViewController.swift

```diff
@@ -111,38 +140,40 @@ extension HomeViewController: UITableViewDataSource {
 			return 130.0
 		case 2 :
 			return 220.0
+		case 7:
```

Comment:
What is case 2 & 7 ? 
Please make an enum and describe about it. Don't using magic number

Reply:
fix complete